lib/build-search.ts

```diff
@@ -32,23 +32,20 @@ import { getSortedWorkflowsData } from "./workflows";
       process.env.ALGOLIA_SEARCH_ADMIN_KEY!
     );
 
-    // Initialize the index with the algolia index name we set up
-    const index = client.initIndex("workflow_specs");
-    
+    const indexName = "workflow_specs";
```

Comment:
In version 4 of `algoliasearch`, you called `initIndex` on a client, which would then use that index for all future calls. Now, you provide the index to those individual calls. See [here](https://www.algolia.com/doc/libraries/javascript/v5/upgrade/#removal-of-initindex) for more info.